Source/Core/Core/PowerPC/JitArm64/JitAsm.cpp

```diff
@@ -95,36 +95,24 @@ void JitArm64::GenerateAsm()
 
   bool assembly_dispatcher = true;
 
-  auto& memory = m_system.GetMemory();
-
   if (assembly_dispatcher)
   {
-    // set the mem_base based on MSR flags
-    LDR(IndexType::Unsigned, ARM64Reg::W28, PPC_REG, PPCSTATE_OFF(msr));
-    FixupBranch physmem = TBNZ(ARM64Reg::W28, 31 - 27);
-    MOVP2R(MEM_REG,
-           jo.fastmem_arena ? memory.GetPhysicalBase() : memory.GetPhysicalPageMappingsBase());
-    FixupBranch membaseend = B();
-    SetJumpTarget(physmem);
-    MOVP2R(MEM_REG,
-           jo.fastmem_arena ? memory.GetLogicalBase() : memory.GetLogicalPageMappingsBase());
-    SetJumpTarget(membaseend);
+    ARM64Reg msr = ARM64Reg::W27;
+    LDR(IndexType::Unsigned, msr, PPC_REG, PPCSTATE_OFF(msr));
```

Comment:
This LDR can be moved back to where it was before this PR.

Reply:
This has now been moved, but not to where it was before this PR. Was that done intentionally, for better codegen?

Reply:
sorry, i moved it back